cachito/web/validation.py

```diff
@@ -94,3 +95,19 @@ def handle_wrapper(request):
                 raise ValidationError(exception.detail)
 
         return handle_wrapper
+
+
+@jsonschema.draft4_format_checker.checks("datetime")
+def datetime_validator(val: str) -> bool:
+    """Validate that datetime fields have the correct format."""
+    if not isinstance(val, str):
+        raise ValidationError(
+            f"'{val}' is not string type to be evaluated as datetime(ISO 8601 format)."
+        )
```

Comment:
> Minor typing nitpick, otherwise LGTM
> 
> Is the CI failing due to the new /sbom endpoint in the master branch?

Yes, I'm working to fix it right now!